db/migrate/20160808002834_add_files.rb

```diff
@@ -0,0 +1,29 @@
+class AddFiles < ActiveRecord::Migration
+  def change
+    create_table "katello_files" do |t|
+      t.timestamps
+      t.string "uuid", :null => false, :limit => 255
+      t.string 'name', :limit => 255
+      t.string 'checksum', :limit => 255
```

Comment:
@jlsherrill locally I have it looking like this:

```
{
id: 6,
name: "katello-ca-consumer-latest.noarch.rpm",
path: "pops/hey/WHAT",
uuid: "9d03acff-e674-4f49-8753-f900e7d22d12",
checksum: "24065bf79b81abd806fbe989edcfffcc1e3b4bb8a02726cbfbc957de3df166fd"
},
```

Do you think the full path would be better there? or just let users and interfaces construct it?